frontend/js/src/user-feed/UserFeed.tsx

```diff
@@ -540,6 +675,34 @@ export default function UserFeedPage() {
         />
       );
     }
+    if (event_type === EventType.THANKS) {
+      const {
+        original_event_id,
+        original_event_type,
+        thanker_id,
+        thanker_username,
+        thankee_id,
+        thankee_username,
+        blurb_content,
```

Comment:
Unused variables here:
```suggestion
        original_event_type,
        thanker_username,
        thankee_username,
```